mmedit/apis/matting_inference.py

```diff
@@ -21,12 +22,14 @@ def init_model(config, checkpoint=None, device='cuda:0'):
     Returns:
         nn.Module: The constructed model.
     """
+    register_all_modules(init_default_scope=True)
     if isinstance(config, str):
         config = mmcv.Config.fromfile(config)
     elif not isinstance(config, mmcv.Config):
         raise TypeError('config must be a filename or Config object, '
                         f'but got {type(config)}')
-    config.model.pretrained = None
```

Comment:
In the new design, pretrain is no longer an item in model. Instead, we will use `init_cfg`.

```
model = dict(
type='xxx'
init_cfg=dict(
type='Pretrain', 
weight='some_path'
)
```
like `https://github.com/open-mmlab/mmediting/blob/dev-1.x/configs/mattors/gca/baseline_r34_4x10_200k_comp1k.py`

Pre-train weight is useless and so we can find `init_cfg` recursively and delete them.

Reply:
done